src/app/app.component.ts

```diff
@@ -33,13 +38,21 @@ export class AppComponent {
   title = 'Task Manager'
 
   isLoggedIn$: Observable<String | null> = this.usersService.isLoggedIn$
-  user$: Observable<User | null> = this.usersService.user$
+  user$: Observable<User | null> | undefined
```

Comment:
why don't you just assign user here instead of constructor, also user should remain as an observable of user or null